src/main/java/org/myrobotlab/service/abstracts/AbstractServo.java

```diff
@@ -610,6 +612,25 @@ public Double moveTo(Double newPos) {
     processMove(newPos, false, null);
     return newPos;
   }
+  
+  /**
+   * incrementally move - takes the servo's current position (could be in the middle of a move)
+   * adds the increment and tells the servo to go to the new location. Returns the new location
+   * if the moveTo was successfully started - otherwise returns null
+   * @param increment
+   * @return
+   */
+  public Double moveIncr(Double increment) {
+    if (increment == null) {
+      log.info("will not move to null position - not moving");
+      return null;
+    }
+    double newPos = getCurrentInputPos() + increment;
```

Comment:
during a speed controlled movement, with a low speed set, this will have a strange behavior.  Does getCurrentInputPos take into account the estimated position of the servo as it's moved at this instance in time?

Reply:
onEncoderData sets currentOutputPos - and the getCurrentInputPos() reverse map calculates the "current" input from the current output ...

Not pretty but - all that was existing before ...

TLDR - yes